egs/vcc2020/README.md

```diff
@@ -0,0 +1,63 @@
+# Recipe for Voice Conversion Challenge 2020 baseline : Cascade ASR + TTS
+
+Official homepage: [http://www.vc-challenge.org/](http://www.vc-challenge.org/)
+
+## Introduction
+
+This recipe describes baseline for the Voice Conversion (VC) Challenge 2020 (VCC2020 for short). VCC2020 contains two tasks. In either tasks, the source speech is always an English speech from a native speaker. Task 1 requires to convert to a English target speaker with a small amount of English parallel training set, and task 2 requires to convert to a non-English (German/Finnish/Mandarin) target speaker with a non-English training set. Task 2 is also referred as cross-lingual VC.
+
+## Method
+
+A naive approach for VC is a cascade of an automatic speech recognition (ASR) model and a text-to-speech (TTS) model. In this recipe we revisit this method under an end-to-end (E2E) framework.
+
+- Specifically, we train a (speaker independent) ASR model, and a separate speaker dependent TTS model for each target speaker. First, the ASR model takes the source English speech as input and output the transcribed text. Then, the TTS model of the target speaker synthesizes the converted speech using the recognition result.
+- Since the size of training set of each target speaker is to limited for E2E TTS learning, we use a pretraining-finetuning scheme to improve the performance.
```

Comment:
```suggestion
- Since the size of the training set of each target speaker is too limited for E2E TTS learning, we use a pretraining-finetuning scheme to improve the performance.
```